DotNet/emitter.js

```diff
@@ -136,16 +138,24 @@
 	if ("parentBranchName" in test.config) addHook('beforeEach', dot_net`eyes.ParentBranchName = ${test.config.parentBranchName};`)
 	if ("hideScrollbars" in test.config) addHook('beforeEach', dot_net`eyes.HideScrollbars = ${test.config.hideScrollbars};`)
 	if ("isDisabled" in test.config) addHook('beforeEach', dot_net`eyes.IsDisabled = ${test.config.isDisabled};`)
+	if ("batch" in test.config) {
+		if ("id" in test.config.batch) {
+			addHook('beforeEach', dot_net`eyes.Batch.Id = ${test.config.batch.id};`)
+		}
+		if ("properties" in test.config.batch) {
+			addHook('beforeEach', dot_net`eyes.Batch.AddProperty(${test.config.batch.properties[0].name}, ${test.config.batch.properties[0].value});`)
+		}
+	}
 	if (("defaultMatchSettings" in test.config) && ("accessibilitySettings" in test.config.defaultMatchSettings)) {
 		let level = `${test.config.defaultMatchSettings.accessibilitySettings.level}`
 		let version = `${test.config.defaultMatchSettings.accessibilitySettings.guidelinesVersion}`
 		addHook('beforeEach', dot_net`AccessibilitySettings settings = new AccessibilitySettings(AccessibilityLevel.` + level + `, AccessibilityGuidelinesVersion.` + version + `);
-        Applitools.Selenium.Configuration configuration = eyes.GetConfiguration();
+        var configuration = eyes.GetConfiguration();
         configuration.SetAccessibilityValidation(settings);
         eyes.SetConfiguration(configuration);`)
 	}
 	if ("browsersInfo" in test.config) {
-		addHook('beforeEach', dot_net`Applitools.Selenium.IConfiguration config = eyes.GetConfiguration();`)
+		addHook('beforeEach', dot_net`var config = eyes.GetConfiguration();`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
		addHook("beforeEach", dot_net`var config = eyes.GetConfiguration();`)
```